instrumentation/instagocb/README.md

```diff
@@ -0,0 +1,110 @@
+# Instana instrumentation of Couchbase SDK v2 (gocb) for Go
```

Comment:
@instana/eng-go  Please review the README and let me know if there's anything else that should be added.

Reply:
Looks pretty good to me, man!
Very detailed and concise. ⭐⭐⭐⭐⭐

Reply:
It looks very detailed and well written. Really good job @nithinputhenveettil !